### PR TITLE
Adding GemInstallPrerelease function, to install the latest prereleas…

### PR DESCRIPTION
**Pull request by lpusok**
…e version of a gem.

### CODE REVIEW

File: command/rubycommand/rubycommand.go [hunk 1]
```diff
@@ -152,21 +152,48 @@ func GemUpdate(gem string) ([]*command.Model, error) {
 	return cmds, nil
 }
 
+func gemInstallCommand(gem, version string, enablePrerelease bool) []string {
+	slice := []string{"gem", "install", gem, "--no-document"}
+	if enablePrerelease {
+		slice = append(slice, "--pre")
```
Comment (godrei) [inline command/rubycommand/rubycommand.go:158]: based on the [gem install docs](https://guides.rubygems.org/command-reference/#gem-install), it should be:

```
-​-[no-]prerelease - Allow prerelease versions of a gem to be installed. (Only for listed gems)
```

File: command/rubycommand/rubycommand.go [hunk 2]
```diff
@@ -152,21 +152,48 @@ func GemUpdate(gem string) ([]*command.Model, error) {
 	return cmds, nil
 }
 
+func gemInstallCommand(gem, version string, enablePrerelease bool) []string {
```
Comment (godrei) [inline command/rubycommand/rubycommand.go:155]: NOTE: it is a bit strange with this function that either `version` or `enablePrerelease` can be used, but not both.

Comment (lpusok) [reply command/rubycommand/rubycommand.go:155]: Did not consider that version range can be used with prerelease option.

File: command/rubycommand/rubycommand.go (hunk 1 above)
Comment (lpusok) [reply command/rubycommand/rubycommand.go]: It worked fine, but updated it.

**Pull request merged**